query/efs/efs_file_system_backup_plan_enabled.sql

```diff
@@ -0,0 +1,17 @@
+select
```

Comment:
For filename, `query/efs/efs_file_system_backup_plan_enabled.sql` -> `query/efs/efs_file_system_automatic_backups_enabled.sql`